packages/components/src/Disclosure/Disclosure.tsx

```diff
@@ -27,35 +24,35 @@ interface DisclosureProps {
    * @default false
    */
   readonly defaultOpen?: boolean;
+
+  readonly onToggle?: (newOpened: boolean) => void;
+  readonly opened?: boolean;
 }
 
 export function Disclosure({
   children,
   title,
   defaultOpen = false,
+  onToggle,
+  opened,
 }: DisclosureProps) {
-  const [isOpen, setOpen] = useState(defaultOpen);
-  const [isMounted, setMount] = useState(false);
-  const [titleRef, { exactWidth }] = useResizeObserver<HTMLDivElement>();
-  const isBelowBreakpoint = exactWidth && exactWidth < Breakpoints.small;
+  const [internalIsOpened, setInternalIsOpen] = useState(defaultOpen || opened);
+  const isOpen = opened !== undefined ? opened : internalIsOpened;
+  const { smallAndUp } = useBreakpoints();
```

Comment:
changed to use `useBreakpoints`

Reply:
I think this is because the `useResizeObserver` uses the deprecated breakpoints [see](https://atlantis.getjobber.com/?path=/docs/hooks-useresizeobserver--docs#the-breakpoints-object) while `useBreakpoints` uses the "new" ones [see](https://atlantis.getjobber.com/?path=/docs/design-breakpoints--docs). I noticed that when one the new breakpoints it was the same size as before it just needed to be on a different viewport size.

CC: @chris-at-jobber

Reply:
Actually after reading the docs for `useResizeObserver` I should revert this change. `useResizeObserver` cares about the size of the actual component and not the viewport. I will undo the change

Reply:
@jdeichert I undid the change in [3adf1e5](https://github.com/GetJobber/atlantis/pull/2088/commits/3adf1e5680ff948e8b23b24aae3b75d469c724a8)